cmd/juju/commands/main.go

```diff
@@ -70,6 +70,44 @@ or your local machine.
 See https://discourse.jujucharms.com/ for ideas, documentation and FAQ.
 `
 
+var usageHelp = `
+Usage: juju [help] <command>
+
+Summary:
+Juju is model & application management software designed to leverage the power
+of existing resource pools, particularly cloud-based ones. It has built-in
+support for cloud providers such as Amazon EC2, Google GCE, Microsoft
+Azure, OpenStack, and Rackspace. It also works very well with MAAS and
+LXD. Juju allows for easy installation and management of workloads on a
+chosen resource pool.
+
+See https://jujucharms.com/docs/stable/help for documentation.
+
+Common commands:
+
+    add-cloud           Adds a user-defined cloud to Juju.
+    add-credential      Adds or replaces credentials for a cloud.
+    add-model           Adds a hosted model.
+    add-relation        Adds a relation between two applications.
+    add-unit            Adds extra units of a deployed application.
+    add-user            Adds a Juju user to a controller.
```

Comment:
If we have this, I don't think `add-user` should be here. I see `add-user` as more of a power user command, one that needs better understanding and limitations than, something you'd reach for right away.